code/game/machinery/newscaster.dm

```diff
@@ -887,6 +887,16 @@ GLOBAL_LIST_EMPTY(allCasters)
 	var/wantedBody
 	var/wantedPhoto
 	var/creationTime
+	var/datum/horoscope/horoscope
+
+/obj/item/newspaper/Initialize(mapload)
+	. = ..()
+	horoscope = new
+
+/obj/item/newspaper/examine(mob/user)
+	. = ..()
+	if(horoscope.check_horoscope(user))
```

Comment:
remove this check and the check_horoscope proc

Reply:
but what if you need to check if you get a new one later

Reply:
check_horoscope does nothing since now you can view the horoscope anytime.

The only purpose of check_horoscope is currently to hide this exame text.